lib/netext/httpext/tracer_test.go

```diff
@@ -232,10 +227,9 @@ func TestCancelledRequest(t *testing.T) {
 		}()
 
 		resp, err := srv.Client().Transport.RoundTrip(req)
-		trail := tracer.Done()
-		if resp == nil && err == nil && len(trail.Errors) == 0 {
-			t.Errorf("Expected either a RoundTrip response, error or trail errors but got %#v, %#v and %#v", resp, err, trail.Errors)
-		}
+		_ = tracer.Done()
+		assert.Nil(t, resp)
+		assert.Error(t, err)
```

Comment:
it's 
```
2020/10/01 15:46:31 http: TLS handshake error from 127.0.0.1:57990: remote error: tls: bad certificate
--- FAIL: TestTracerError (0.04s)
        Error Trace:    tracer_test.go:208
        Error:          Received unexpected error:
                        x509: certificate signed by unknown authority
        Test:           TestTracerError
```
I am more okay to get the `errorCode` out of it and check that? as otherwise this messages sometimes change and we need to the whole dance with changing the error message or using `includes`